html/pfappserver/root/src/views/Configuration/provisioners/_components/FormTypeGoogleWorkspaceChromebook.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <base-form
+    :form="form"
+    :meta="meta"
+    :schema="schema"
+    :isLoading="isLoading"
+  >
+    <form-group-identifier namespace="id"
+      :column-label="$i18n.t('Provisioning ID')"
+      :disabled="!isNew && !isClone"
+    />
+
+    <form-group-description namespace="description"
+      :column-label="$i18n.t('Description')"
+    />
+
+    <form-group-enforce namespace="enforce"
+      :column-label="$i18n.t('Enforce')"
+      :text="$i18n.t('Whether or not the provisioner should be enforced. This will trigger checks to validate the device is compliant with the provisioner during RADIUS authentication and on the captive portal.')"
+    />
+
+    <form-group-auto-register namespace="autoregister"
+      :column-label="$i18n.t('Auto register')"
+      :text="$i18n.t('Whether or not devices should be automatically registered on the network if they are authorized in the provisioner.')"
+    />
+
+    <form-group-apply-role namespace="apply_role"
+      :column-label="$i18n.t('Apply role')"
+      :text="$i18n.t('When enabled, this will apply the configured role to the endpoint if it is authorized in the provisioner.')"
+    />
+
+    <form-group-role-to-apply namespace="role_to_apply"
+      :column-label="$i18n.t('Role to apply')"
+      :text="$i18n.t(`When 'Apply role' is enabled, this defines the role to apply when the device is authorized with the provisioner.`)"
+    />
+
+    <form-group-category namespace="category"
+      :column-label="$i18n.t('Roles')"
+      :text="$i18n.t('Nodes with the selected roles will be affected.')"
+    />
+
+        <form-group-oses namespace="oses"
+          :column-label="$i18n.t('OS')"
+          :text="$i18n.t('Nodes with the selected OS will be affected.')"
+        />
+
+   <form-group-non-compliance-security-event namespace="non_compliance_security_event"
+     :column-label="$i18n.t('Non compliance security event')"
+     :text="$i18n.t('Which security event should be raised when non compliance is detected.')"
+   />
+
+    <form-group-expires-in namespace="expires_in"
+      :column-label="$i18n.t('Token Expires In')"
+      :text="$i18n.t('The amount of seconds of the suggested expiration time.')"
```

Comment:
Do we really need to make the user configure this parameter ?
Also, it doesn't seem to have a default value so I'd be pretty lost on what to put there